instrumentation/pulsar/pulsar-2.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pulsar/v2_8/telemetry/PulsarSingletons.java

```diff
@@ -87,24 +99,28 @@ private static Instrumenter<PulsarBatchRequest, Void> createConsumerBatchReceive
             createMessagingAttributesExtractor(getter, MessageOperation.RECEIVE))
         .addAttributesExtractor(
             ServerAttributesExtractor.create(new PulsarNetClientAttributesGetter()))
-        .setEnabled(ExperimentalConfig.get().messagingReceiveInstrumentationEnabled())
```

Comment:
spring kafka instrumentation also produces telemetry with span links for the batch receive scenario that can't be modeled with just parent child relationships